files/clappr.playback-name/update.json

```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "github",
+  "name": "clappr-playback-name-plugin",
+  "repo": "barbosa/clappr-playback-name-plugin",
+  "files": {
+    "basePath": "dist/",
+    "include": ["playbackname.js", "playbackname.min.js"]
```

Comment:
1. Remove the unminified file from here too
2. There is not dist/ directory in your repo.